portal/integrated_windows_authentication/README.md

```diff
@@ -0,0 +1,46 @@
+# Integrated windows authentication
+
+Use Windows credentials to access services hosted on a portal secured with Integrated Windows Authentication (IWA).
+
+![](IntegratedWindowsAuthentication.png)
+
+## Use case
+
+IWA, which is built into Microsoft Internet Information Server (IIS), allows users to easily authenticate for different services using their domain's Windows credentials. This simplifies logging in to services that are provided through an intranet environment, as only a single username and password combination are required for the applications that support IWA.
+
+## How to use the sample
+
+Select "Search Public Portal" to return all portal results hosted on www.arcgis.com (without requiring IWA authentication) or enter a URL to your IWA-secured portal, and select "Search IWA Secured Portal" to search for web maps stored there. If the latter was chosen, you will be prompted for a username (including domain, such as username@DOMAIN or domain\username), and password. If authentication is successful, portal item results will display in the list. Select a web map item to display it in the map view.
+
+## How it works
+
+1.  Set up an `AuthenticationChallengeHandler` that challenges the user for authentication, such as the custom handler created for this sample: `IWAChallengeHandler`.
+2.  Set the `AuthenticationManager` to use the created challenge handler, `.setAuthenticationChallengeHandler(new IWAChallengeHandler());`.
+3.  Create a new `Portal` from a portal URL and a boolean indicating if it is a secured resource, `new Portal (url, true)`.
+4.  Load the portal, `Portal.loadAsync()`. This will trigger the challenge handler to prompt for a username and password. Depending on whether authentication is successful or not, access to the portal will be granted.
```

Comment:
Load the portal. The authentication challenge handler will prompt the user for credentials. The portal will load successfully with valid credentials.